src/main/kotlin/de/bund/digitalservice/useid/tracking/WebRequests.kt

```diff
@@ -0,0 +1,24 @@
+package de.bund.digitalservice.useid.tracking
+
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClient
+import reactor.core.publisher.Mono
+import java.net.URI
+
+@Service
+class WebRequests(private val client: WebClient) {
+
+    fun POST(url: String): Mono<ResponseEntity<Void>> {
+        return client
+            .post()
+            .uri(URI(url))
+            .retrieve()
+            .toBodilessEntity()
+            .onErrorReturn(
+                // everything but 200 will be caught and responded with 500
+                ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(null)
+            )
```

Comment:
Why do we need this? The response which is returned to the caller is only checked for success (200 OK) anyways. Can be removed imho.

Reply:
Regardless of what the caller does, imo the function should let the caller know if something went wrong

Reply:
If we just return the response which we received from the API, the caller can handle it in the desired way. Why mapping it here (and losing information)?